recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "pyreportjasper" %}
-{% set version = "1.0.2" %}
+{% set version = "2.1.0" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
As far as I remember, conda-forge discourages using github links instead of pypi

Reply:
I am waiting for the acceptance of a request to increase the upload limit for the pypi and until today I have not received a reply, I have not been able to send the new version yet to the pypi.